Entities/AEmitter.h

```diff
@@ -562,6 +562,42 @@ ClassInfoGetters
 
 	void SetEmitCountLimit(long newValue) { m_EmitCountLimit = newValue; }
 
+	/// <summary>
+	/// Gets this AEmitter's Emission Sound. Ownership is not transferred!
+	/// </summary>
+	/// <returns>The SoundContainer for this AEmitter's Emission Sound.</returns>
+	SoundContainer* GetEmissionSound() const { return m_EmissionSound; }
+
+	/// <summary>
+	/// Sets this AEmitter's Emission Sound.
+	/// </summary>
+	/// <param name="newSound">The new SoundContainer for this AEmitter's Emission Sound</param>
```

Comment:
See comments in ACrab.h about `*` placement, method comment capitalization and `.`s at the end of method comments.